FWCore/Services/plugins/SimpleMemoryCheck.cc

```diff
@@ -350,6 +355,22 @@ namespace edm {
 
       openFiles();
 
+      if (sampleEveryNSeconds_ > 0) {
+        if (oncePerEventMode_) {
+          throw cms::Exception("BadConfigOptions")
```

Comment:
Should this use
```suggestion
          throw cms::Exception("Configuration")
```
or
```suggestion
          throw edm::Exception(edm::errors::Configuration)
```
?